europe/ISL

```diff
@@ -0,0 +1,312 @@
+# LCC ISN2016 (epoch 2016.5, IGS2014, GRS80):  The primary system  
+#----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+<ISN2016_LAM>  +proj=lcc  +lat_1=64.25 +lat_2=65.75 +lat_0=65 +lon_0=-19 +x_0=2700000 +y_0=300000  +ellps=GRS80   +units=m +no_defs
+
+#----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+<gie>
+operation   init=ISL:ISN2016_LAM
+tolerance   0.1 mm
+accept      -17 66
+expect      2790800.4329    412932.6164 
+</gie>
+
+
+#-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# LCC ISN2004 (epoch 2004.6, IGb00, GRS80):  The other systems 
+#-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+<ISN2004_LAM>   +proj=lcc  +lat_1=64.25 +lat_2=65.75 +lat_0=65 +lon_0=-19 +x_0=1700000 +y_0=300000  +ellps=GRS80   +units=m +no_defs
+
+#-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+<gie>
+operation   init=ISL:ISN2004_LAM
+tolerance   0.1 mm
+accept      -17 66
+expect      1790800.4329    412932.6164
+</gie>
+
+
+#------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# LCC ISN93 (epoch 1993.6, ITRF93, GRS80):  The other systems 
+#------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+<ISN93_LAM>     +proj=lcc  +lat_1=64.25 +lat_2=65.75 +lat_0=65 +lon_0=-19 +x_0=500000  +y_0=500000  +ellps=GRS80   +units=m +no_defs
+
+#------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+<gie>
+operation   init=ISL:ISN93_LAM
+tolerance   0.1 mm
+accept      -17 66
+expect      590800.4329    612932.6164
+</gie>
+
+
+#-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+# LCC Hj�rsey1955 (ellipsoid: International 1924):  The historical systems 2D
```

Comment:
encoding issue ? I assume we probably want UTF-8

Reply:
I have changed the ö to o.